.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
   directories:
   - node_modules
 script:
-  - npm run test src/App.test.js
+  - npm run ci
```

Comment:
@Spetastian Thank you :+1: good catch.

Reply:
Our tests will appear to fail because of our empty *.test.js files, but that's fine for now; we can either delete them or add tests really soon

Reply:
The latter 😄

Reply:
lol good idea